package.json

```diff
@@ -69,7 +69,7 @@
     "helmet": "^3.23.1",
     "http-errors": "~1.6.3",
     "jwt-decode": "^3.1.2",
-    "knex": "^0.21.1",
+    "knex": "^0.95.13",
```

Comment:
Is this knex update safe?

Reply:
good question, im gonna look into that

Reply:
documentation says knex will work backwards but not forwards, in terms of updates, so we should be good